### PR TITLE
fix(nvmf): fixing duplicate QID error

### PR DESCRIPTION
**Pull request by dsavitskiy**
* Updating to the SPDK with the fix
* A test for many parallel rebuilds added

### CODE REVIEW

**Comment by dsavitskiy** [comment]
bors try

File: io-engine/tests/nexus_rebuild_parallel.rs [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        .add_container_bin(
+            "ms_0",
+            Binary::from_dbg("io-engine")
+                .with_args(vec!["-l", "1,2", "-Fcolor,compact,nodate,host"])
+                .with_bind("/tmp", "/host/tmp"),
+        )
+        .add_container_bin(
+            "ms_1",
+            Binary::from_dbg("io-engine")
+                .with_args(vec!["-l", "2,3", "-Fcolor,compact,nodate,host"])
```
Comment (dsharma-dc) [inline io-engine/tests/nexus_rebuild_parallel.rs]: core id 2 is overlapping here with first io-engine instance.

**Review by dsharma-dc — APPROVED** [review]

**Review by hrudaya21 — APPROVED** [review]

File: io-engine/tests/nexus_rebuild_parallel.rs (hunk 1 above)
Comment (dsavitskiy) [reply io-engine/tests/nexus_rebuild_parallel.rs]: Thanks, I will fix.

**Comment by dsavitskiy** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
@dsavitskiy you haven't merged the spdk PR?

**Comment by tiagolobocastro** [comment]
Make sure the commit hash matches...

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**